plugins/arrow-flight-rpc/src/test/resources/org/opensearch/bootstrap/test.policy

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+grant codeBase "${codebase.arrow-flight-rpc:arrow-memory-shaded}" {
```

Comment:
I think this policy needs a cleanup

Reply:
this is actually no more needed. I got it working just with plugin's security policy. I will publish the changes in the next iteration

Reply:
done